sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline.py

```diff
@@ -2800,18 +2809,84 @@ def register_both_output():
         pipeline.outputs.pipeine_a_output.name = "p1_output"
         pipeline.outputs.pipeine_a_output.version = "v1"
         pipeline.settings.default_compute = "cpu-cluster"
-        pipeline_job = client.jobs.create_or_update(pipeline)
-        client.jobs.stream(pipeline_job.name)
+        pipeline_job = assert_job_cancel(pipeline, client)
 
-        def check_name_version_and_register_succeed(output, output_name, output_version):
-            assert output.name == output_name
-            assert output.version == output_version
-            assert client.data.get(name=output_name, version=output_version)
+        check_name_and_version(pipeline_job.outputs.pipeine_a_output, "p1_output", "v1")
+        check_name_and_version(pipeline_job.jobs["node_2"].outputs.component_out_path, "n2_output", "v1")
+        assert pipeline_job.jobs["node_3"].outputs.component_out_path.name == "n3_output"
+        check_name_and_version(pipeline_job.jobs["sub_node"].outputs.sub_pipeine_a_output, "sub_pipeline", "v1")
 
-        check_name_version_and_register_succeed(pipeline_job.outputs.pipeine_a_output, "p1_output", "v1")
-        check_name_version_and_register_succeed(
-            pipeline_job.jobs["node_2"].outputs.component_out_path, "n2_output", "v1"
-        )
-        check_name_version_and_register_succeed(
-            pipeline_job.jobs["sub_node"].outputs.sub_pipeine_a_output, "sub_pipeline", "v1"
-        )
+    @pytest.mark.skip("https://msdata.visualstudio.com/Vienna/_workitems/edit/2245690")
```

Comment:
Glad to see we have a task for this, remember to reopen this when the task finishes.

Reply:
Looks like the server task is already done now. should be included in release today.

Reply:
I skip this test because we meet Internal Server Error now. I use this task to re-open this test https://msdata.visualstudio.com/Vienna/_workitems/edit/2269646/